packages/azure-monitoring/src/main/index.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-node'
+import { getNodeAutoInstrumentations, InstrumentationConfigMap } from '@opentelemetry/auto-instrumentations-node'
+import { PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics'
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions'
+import { Resource } from '@opentelemetry/resources'
+import { SeverityNumber } from '@opentelemetry/api-logs'
+
+import { AzureMonitorMetricExporter, AzureMonitorTraceExporter } from '@azure/monitor-opentelemetry-exporter'
+
+import process from 'process'
+import type { AzureMonitoringOptions } from './types/AzureMonitoringOptions'
+
+const defaultInstrumentationOptions: InstrumentationConfigMap = {
+  '@opentelemetry/instrumentation-bunyan': { logSeverity: SeverityNumber.INFO },
+}
+
+const setUpAzureMonitor = ({ applicationInfo, instrumentationOptions = {} }: AzureMonitoringOptions) => {
+  const exporter = new AzureMonitorTraceExporter()
+
+  const sdk = new NodeSDK({
+    resource: new Resource({
+      [ATTR_SERVICE_NAME]: applicationInfo.applicationName,
+      [ATTR_SERVICE_VERSION]: applicationInfo.buildNumber,
+    }),
+    traceExporter: exporter,
```

Comment:
not entirely sure why trace exporter is referenced here and in the `BatchSpanProcessor`